lib/SIL/IR/SILFunction.cpp

```diff
@@ -138,44 +134,36 @@ static FunctionParseFn parseFunction = nullptr;
 static FunctionCopyEffectsFn copyEffectsFunction = nullptr;
 static FunctionGetEffectFlagsFn getEffectFlagsFunction = nullptr;
 
-SILFunction::SILFunction(SILModule &Module, SILLinkage Linkage, StringRef Name,
-                         CanSILFunctionType LoweredType,
-                         GenericEnvironment *genericEnv,
-                         Optional<SILLocation> Loc, IsBare_t isBareSILFunction,
-                         IsTransparent_t isTrans, IsSerialized_t isSerialized,
-                         ProfileCounter entryCount, IsThunk_t isThunk,
-                         SubclassScope classSubclassScope,
-                         Inline_t inlineStrategy, EffectsKind E,
-                         const SILDebugScope *DebugScope,
-                         IsDynamicallyReplaceable_t isDynamic,
-                         IsExactSelfClass_t isExactSelfClass,
-                         IsDistributed_t isDistributed)
+SILFunction::SILFunction(
```

Comment:
Deleted loc here, around #line133 and #line159
The rest of the changes is formatting

Reply:
Normally people recommend to only format the lines you changed because makes it easier for the reviewer and keep the commit history for the rest of the file cleaner =] 
You can use either `git-clang-format` tool that is on `${PATH_TO_SWIFT}/llvm-project/clang/tools/clang-format` and you can add it to your path, and run on the git directory after `git add .`
Or `clang-format -lines 1630:1660 lib/Sema/CSFix.h -I` for example to run at lines in specific.